odoo/netsvc.py

```diff
@@ -66,7 +78,24 @@ def emit(self, record):
             levelname = logging.getLevelName(record.levelno)
 
             val = ('server', ct_db, record.name, levelname, msg, record.pathname, record.lineno, record.funcName)
-            cr.execute("""
+
+            try:
+                if self._support_metadata and record.levelno >= logging.WARNING:
+                    metadata = {}
+                    if module.current_test:
+                        try:
+                            metadata['test'] = module.current_test.get_log_metadata()
+                        except:
+                            pass
+                    val = (*val, json.dumps(metadata))
+                    cr.execute(f"""
+                        INSERT INTO ir_logging(create_date, type, dbname, name, level, message, path, line, func, metadata)
+                        VALUES (NOW() at time zone 'UTC', %s, %s, %s, %s, %s, %s, %s, %s, %s)
+                    """, val)
+                    return
+            except Exception as e:
+                pass
```

Comment:
Why is this wrapped in a try except? What kind of exception do we expect from here?